content/CORE/Tasks/Rsync.md

```diff
@@ -5,83 +5,83 @@ weight: 30
 
 {{< toc >}}
 
-Data often needs to be copied to another system for backup or when migrating to a new system.
+You often need to copy data to another system for backup or when migrating to a new system.
 A fast and secure way of doing this is by using [rsync](https://rsync.samba.org/).
-These instructions assume that TrueNAS systems are being used for both sides of the rsync task: *host* and *remote*.
+These instructions assume that both sides of the rsync task, host and remote, use a TrueNAS systems.
```

Comment:
These instructions assume that both sides of the rsync task, host and remote, use TrueNAS systems.